doc/news/_preparation_next_release.md

```diff
@@ -120,7 +120,22 @@ you up to date with the multi-language support provided by Elektra.
 
 ## Documentation
 
-- <<TODO>>
+- The [Markdown Link Converter](https://master.libelektra.org/doc/markdownlinkconverter) now uses the style
+
+  ```
+  filename:line:0
+  ```
+
+  instead of
+
+  ```
+  filename|line col 0|
+  ```
+
+  to show the location data for broken links. This has the advantage, that certain tools such as [TextMate](https://macromates.com) are
```

Comment:
As discussed: please write here that this is a default format for gcc, clang, ...